### ISSUE TITLE
Incorrect version for node type definitions

### ISSUE THREAD

**Issue by eugirdor** [castlabs/electron-releases#16]
The package.json has type definitions for Node.js 7, I believe coming from here:

https://github.com/castlabs/electron-releases/blob/906255316e95dcc1329d55a9ae36de9a91c803c6/package.json#L14

But Electron 1.8.1+ uses Node.js 8

https://electronjs.org/releases#1.8.1

This causes warnings when you try to use this package with an application that has a dependency on the Node.js 8 type definitions.

```
error TS4090: Conflicting definitions for 'node' found at '.../node_modules/@types/node/index.d.ts' and '.../node_modules/electron/node_modules/@types/node/index.d.ts'. Consider installing a specific version of this library to resolve the conflict.
```

**Comment by khwaaj** [comment]
Good catch, I'll make sure it gets fixed for 1.8.8 (and other future releases).

**Comment by khwaaj** [comment]
If you have not seen it already I published releases based on Electron 3.0.0-beta.9 and 1.8.8, both of which now has the same node version specified as the original release.

**Comment by eugirdor** [comment]
Just tried it out, looks good.  Thank you for the quick response.
